src/main/java/fourstep/logic/Car.java

```diff
@@ -0,0 +1,53 @@
+package fourstep.logic;
+
+
+import java.util.ArrayList;
+
+public class Car {
+
+    private static final int MOVE_THRESHOLD = 4;
+
+    private String name;
```

Comment:
```suggestion
    private final String name;
```

변하지 않는 값에는 `final` 키워드를 붙이면 좋겠죠?